manual/simulationcell.tex

```diff
@@ -26,17 +27,18 @@ \section{Specifying the simulation cell}
 
 An example of a \texttt{simulationcell} block is given below:
 \begin{shade}
- <simulationcell>
- <parameter name="lattice">
-                 3.80000000       0.00000000       0.00000000
-                 0.00000000       3.80000000       0.00000000
-                 0.00000000       0.00000000       3.80000000
-         </parameter>
-         <parameter name="bconds">
-            p p p
-         </parameter>
-         <parameter name="LR_dim_cutoff">20</parameter>
-</simulationcell>
+  <simulationcell>
+    <parameter name="lattice">
+      3.80000000       0.00000000       0.00000000
+      0.00000000       3.80000000       0.00000000
+      0.00000000       0.00000000       3.80000000
+    </parameter>
+    <parameter name="bconds">
+       p p p
+    </parameter>
+    <!--parameter name="vacuum"> 1.0 </parameter-->
```

Comment:
I don't think this is clear, being put in here with an example that won't use the 'vacuum' option even if it is uncommented.   It should be put in a separate example with p p n or p n n boundary conditions.

Reply:
You are right. I will add an example in the vacuum section.